### PR TITLE
{AKS} az aks to use 2021-02-01 API.

### PR DESCRIPTION
**Pull request by JunSun17**
**Description**

[AKS] az aks to use 2021-02-01 API.

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: src/azure-cli/setup.py [hunk 1]
```diff
@@ -76,7 +76,7 @@
     'azure-mgmt-containerinstance~=1.4',
     'azure-mgmt-containerregistry==3.0.0rc16',
     'azure-mgmt-cosmosdb~=3.0.0',
-    'azure-mgmt-containerservice~=9.4.0',
+    'azure-mgmt-containerservice~=11.1.0',
```
Comment (yungezz) [inline src/azure-cli/setup.py:79]: pls rerun live test and re-recording tests, to avoid regression and update tests.

Comment (JunSun17) [reply src/azure-cli/setup.py:79]: test recording updated.

**Comment by JunSun17** [comment]
@andyzhangx This is the 2021-03-01 API PR.

**Review by yungezz — APPROVED** [review]
pls rebase from dev

**Comment by JunSun17** [comment]
> pls rebase from dev

rebased.

**Comment by yungezz** [comment]
@JunSun17 PR title convention https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests. SInce this will not impact command interface, you could use `{AKS}`

**Comment by JunSun17** [comment]
> @JunSun17 PR title convention https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests. SInce this will not impact command interface, you could use `{AKS}`

Sure, changed to {AKS}

**Comment by JunSun17** [comment]
@fengzhou-msft The PR passed all tests, can you help to merge it?

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**